app/policies/post_policy.rb

```diff
@@ -0,0 +1,17 @@
+class PostPolicy < ApplicationPolicy
```

Comment:
There no one unit spec. Feature specs in not enough. You should add unit specs that will cover different situations.